common/lib/xmodule/xmodule/x_module.py

```diff
@@ -1609,6 +1609,18 @@ def filestore(self):
         )
         return self.resources_fs
 
+    @property
+    def node_path(self):
+        """
+        Path to node_modules. Doesn't seem to be used by any ModuleSystem dependent core XBlock anymore.
+
+        Deprecated.
+        """
+        warnings.warn(
+            'node_path is deprecated. Please use other methods of finding the node_modules location.',
+            DeprecationWarning, stacklevel=3
```

Comment:
@jristau1984: After this merges and deploys, please try searching for this log message in Splunk. The only use I recall that this functionality ever had was for some early Berkeley courses that had to be explicitly put on an allow list to directly call node in some of their problem code. My recollection is that this functionality is long gone and those courses are not even accessible any longer (as they were Old Mongo courses). But if there are other instances in the code that try to use the node_path in this way, it should show up in the logs.